core/core-dotspacemacs.el

```diff
@@ -162,6 +162,9 @@ size to make separators look not too crappy.")
 (defvar dotspacemacs-ex-substitute-global nil
   "If non nil, inverse the meaning of `g' in `:substitute' Evil ex-command.")
 
+(defvar dotspacemacs-override-evil-folding nil
```

Comment:
Maybe a `dotspacemacs-folding-method` taking as option `'evil` or `'origiami` will be easier to extend to eventual future methods.